bibtex-actions.el

```diff
@@ -417,10 +421,67 @@ TEMPLATE."
 ;; something equivalent.
 (defun bibtex-actions-get-key-org-cite ()
   "Return key at point for org-cite citation-reference."
-  (when (eq major-mode 'org-mode)
-    (let ((elt (org-element-context)))
-      (if (eq (car elt) 'citation-reference)
-          (org-element-property :key elt)))))
+  (when-let (((eq major-mode 'org-mode))
```

Comment:
I think that should be `when-let*`, since the second let binding depends on the buffer being in org mode, which is checked in the first binding.

Reply:
`when-let` evaluates each binding in turn and stops if a binding value is nil, i.e. the second let binding is not evaluated in non org-mode buffers.

Reply:
You are right! Expanding the macro shows it is using `let*` under the hood. Didn't know this; I always used `when-let*` because I took it to be similar to `let` vs. `let*`. Good to know, thanks!